README.md

```diff
@@ -18,23 +18,26 @@ const Persist = require('persist-store')
 // custom persisters should implement promises
 
 class LogIt {
-  save(name, value) {
+  save(value) {
     return Promise.resolve('this does nothing')
   }
-  load(name) {
-    return Promise.resolve("shh, act like i loaded something:" + name)
+  load() {
+    return Promise.resolve('shh, act like i loaded something')
   }
 }
 
 // example options
 const persist = new Persist([
     {
       type: 'local',
-      path: './data'
+      path: './data',
```

Comment:
I was doing that originally and I cooould do that, but the big issue is that if the directory doesn't exist, it creates it. 

I know I could split it and all and but eh, figured more deliberate was better. Either is cool with me.